redash/utils.py

```diff
@@ -95,9 +95,9 @@ def default(self, o):
         if isinstance(o, decimal.Decimal):
             return float(o)
 
-        if isinstance(o, datetime.date):
+        if isinstance(o, datetime.date) or isinstance(o, datetime.datetime):
```

Comment:
Mongo (and other runners) sometimes return a datetime.datetime object so I thought instead of having it in the mongodb json serializer I'll add it here.

datetime.date != datetime.datetime :-)